index.js

```diff
@@ -99,7 +103,10 @@ function resolveUrlLoader(content, sourceMap) {
   if (useMap) {
 
     // source-map sources seem to be relative to the file being processed
-    absoluteToRelative(reworked.map.sources, filePath);
+    absoluteToRelative(reworked.map.sources, path.resolve(filePath, sourceRoot));
```

Comment:
Obviously you had a problem with this. Are you able to give me a short explanation on the github issue.